styles/styles.css

```diff
@@ -0,0 +1,1844 @@
+  padding: 0;
+}
+.dog-card p {
+  text-align: inherit;
+  align-items: center;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+}
+.pricing-listing {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0;
+  margin-left: 0;
+}
+.total-listing { 
+  font-weight: bold;
+  font-size: 1rem;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.price-listing {
+  font-size: 1rem;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.flex-dog-button {
+  background-color: #88cc00;
+  color: white;
+  font-family:Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 1.3rem;
+  font-weight: bold;
+  width: 90%;
+  height: 3rem;
+  margin: 0 0 1rem 1rem;
+  align-items: center;
+  border: 0;
+}
+/*--End Dog Card Information--*/
+/*--dog-listing.index.html--*/
+.main-dog-list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+/*--index container--*/
+.dog-list-container {
+  width: 66%;
+  display: flex;
+  height: 100%;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 80%;
+  padding-top: 10rem;
+  gap: 1rem;
+  margin-left: 13%;;
+}
+.dog-listing-title {
+  color:#2f4f4f;
+  width: 100%;
+  font-size: 2.2rem;
+  margin: 0;
+  font-family:Verdana, Geneva, Tahoma, sans-serif;
+}
+.dog-name {
+  color:#2f4f4f;
+  font-family:Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 1.5em;
+}
+/*--Start Dog-blog.html--*/
+.blog-adoptadog-h1 {
+  color:#2f4f4f;
+  padding: 3rem;
+  padding-left: 0;
+  font-size: 2rem;
+  margin-left: 13%;
+}
+main {
+  display: flex;
+  flex-direction: row;
+}
+.container-blog {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+  margin-left: 13%;
+  padding-left: 0;
+  /*padding: 5rem;*/
+}
+.blog-item {
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 7rem;
+  width: 90%;
+}
+.blog-content {
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+  height: 15rem;
+  padding-left: 2rem;
+  padding-bottom: 2rem;
+  padding-top: 0;
+  margin: 0;
+}
+.blog-topic {
+  color:#2f4f4f;
+  width: 100%;
+  margin-top: 0;
+  padding-top: 0;
+}
+/*--End Dog-blog.html page--*/
+/*--Checkout.html form--*/
+.form-main {
+  padding-left: 10%;
+}
+.infomation {
+  line-height: 1.4;
+  display: flex;
+  flex-direction: column;
+  height: 45%;
+}
+.form-group-1 {
+  margin-bottom: 1em;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+.title-adoptadog {
+  color:#2f4f4f;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.form-group-2 {
+  margin-bottom: 1em;
+  padding: 10px;
+}
+.first-time {
+  padding: 0.5rem;
+  align-self: flex-start;
+}
+.pickup-section {
+  padding: 2rem 1rem 1rem 0;
+}
+.pickup {
+  width: 25%;
+  height: 2rem;
+  border: 1px solid #000000;
+}
+.dogs-adopted {
+  color:#2f4f4f;
+  font-size: 1.5rem;
+  font-weight: bold;
+  padding-bottom: 1rem;
+}
+.adopt-final{
+  width: 10%;
+}
+.adopted-dogs {
+  display: flex;
+}
+.adopted-dogs img {
+  width: 10%;
```

Comment:
Whenever you set a width, set a height of auto so the image keeps it's aspect ratio.